lib/pbench/server/__init__.py

```diff
@@ -186,7 +187,7 @@ def COMMIT_ID(self) -> str:
 
     @property
     def rest_uri(self) -> str:
-        return self._get_conf("pbench-server", "rest_uri")
+        return self.conf.get("pbench-server", "rest_uri")
```

Comment:
And, you're sure that you want to call `self.conf.get()` rather than `self.get()`?

I believe that they both do the same thing, but using the latter feels more "object-oriented" to me.

Ditto line 249 (et al.?).

Reply:
I'd like to take care of further changes like this in PR #3192 or a derivative.